test/groovy/org/ods/orchestration/BuildStageSpec.groovy

```diff
@@ -0,0 +1,99 @@
+package org.ods.orchestration
+
+import org.ods.PipelineScript
+import org.ods.orchestration.scheduler.LeVADocumentScheduler
+import org.ods.orchestration.usecase.JUnitTestReportsUseCase
+import org.ods.orchestration.usecase.JiraUseCase
+import org.ods.orchestration.util.MROPipelineUtil
+import org.ods.orchestration.util.Project
+import org.ods.services.JenkinsService
+import org.ods.services.ServiceRegistry
+import org.ods.util.ILogger
+import org.ods.util.IPipelineSteps
+import org.ods.util.Logger
+import org.ods.util.PipelineSteps
+import util.SpecHelper
+
+import static util.FixtureHelper.createProject
+
+class BuildStageSpec extends SpecHelper {
+    Project project
+    BuildStage buildStage
+    IPipelineSteps steps
+    PipelineScript script
+    MROPipelineUtil util
+    JiraUseCase jira
+    JUnitTestReportsUseCase junit
+    JenkinsService jenkins
+    LeVADocumentScheduler levaDocScheduler
+    ILogger logger
+
+    def phase = MROPipelineUtil.PipelinePhases.BUILD
+
+    def setup() {
+        script = new PipelineScript()
+        steps = Mock(PipelineSteps)
+        levaDocScheduler = Mock(LeVADocumentScheduler)
+        project = Spy(createProject())
+        util = Mock(MROPipelineUtil)
+        jira = Mock(JiraUseCase)
+        junit = Mock(JUnitTestReportsUseCase)
+        jenkins = Mock(JenkinsService)
+        logger = new Logger(script, !!script.env.DEBUG)
```

Comment:
This is a test, why initialize with !!script.env.DEBUG?

Reply:
C&P from other test. Updated in new commit